ess/src/main/java/org/eclipse/tractusx/ess/notification/mock/MockedNotificationReceiverEndpoint.java

```diff
@@ -0,0 +1,108 @@
+@RequiredArgsConstructor
+@Validated
+@Hidden
+public class MockedNotificationReceiverEndpoint {
+
+    private final EdcDiscoveryMockConfig edcDiscoveryMockConfig;
+    private final EdcSubmodelFacade edcSubmodelFacade;
+
+    @PostMapping("/receive")
+    public void receiveNotification(final @Valid @RequestBody EdcNotification notification) throws EdcClientException {
+        final Optional<String> incidentBpn = Optional.ofNullable(notification.getContent().get("incidentBpn"))
+                                                     .map(Object::toString);
+
+        if (incidentBpn.isPresent()) {
+            final String bpn = incidentBpn.get();
+
+            if (edcDiscoveryMockConfig.getMockEdcResult().containsKey(bpn)) {
+                final SupplyChainImpacted supplyChainImpacted = edcDiscoveryMockConfig.getMockEdcResult().get(bpn);
+
+                final String notificationId = UUID.randomUUID().toString();
+                final String senderEdc = ServletUriComponentsBuilder.fromCurrentRequest().build().toUriString();
+                final String senderBpn = notification.getHeader().getRecipientBpn();
+                final String recipientBpn = notification.getHeader().getSenderBpn();
+                final String recipientUrl = notification.getHeader().getSenderEdc();
+                final Map<String, Object> notificationContent = Map.of("result", supplyChainImpacted.getDescription());
+
+                final EdcNotification edcRequest = edcRequest(notificationId, senderEdc, senderBpn, recipientBpn, notificationContent);
+
+                final var response = edcSubmodelFacade.sendNotification(recipientUrl, "notify-request-asset", edcRequest);
+                if (!response.deliveredSuccessfully()) {
+                    throw new EdcClientException("EDC Provider did not accept message with notificationId " + notificationId);
+                }
+            } else {
+                throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "BPN " + bpn + " not handled.");
+            }
+
+        } else {
+            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Malformed EDC Notification - content without incidentBPN key.");
+        }
+    }
+
+    private EdcNotification edcRequest(final String notificationId, final String senderEdc, final String senderBpn, final String recipientBpn,
+            final Map<String, Object> content) {
+        final EdcNotificationHeader header = EdcNotificationHeader.builder()
+                                                                 .notificationId(notificationId)
+                                                                 .senderEdc(senderEdc)
+                                                                 .senderBpn(senderBpn)
+                                                                 .recipientBpn(recipientBpn)
+                                                                 .replyAssetId("ess-response-asset")
```

Comment:
No re-reply necessary for a reply :)

```suggestion
                                                                 .replyAssetId("")
```